webui/react/src/pages/ExperimentDetails/TrialsComparisonModal.tsx

```diff
@@ -41,37 +40,30 @@ type TableProps = XOR<
 
 type ModalProps = TableProps & {
   onCancel: () => void;
-  visible: boolean;
 };
 
-const TrialsComparisonModal: React.FC<ModalProps> = ({
+const TrialsComparisonModalComponent: React.FC<ModalProps> = ({
   onCancel,
-  visible,
   ...props
 }: ModalProps) => {
-  const resize = useResize();
-  const {
-    themeSettings: { className: themeClass },
-  } = useTheme();
-
   useEffect(() => {
     if (props.trialIds?.length === 0 || props.trials?.length === 0) onCancel();
   }, [onCancel, props.trialIds?.length, props.trials?.length]);
 
   return (
     <Modal
-      centered
-      footer={null}
-      open={visible}
-      style={{ height: resize.height * 0.9 }}
+      submit={{
+        handleError: () => {},
+        handler: () => {},
+        onComplete: onCancel,
+        text: 'Ok',
```

Comment:
I think this one should be Close

Reply:
@mapmeld let me know if you had something else in mind, but I see an issue where we don't have "an informational / Close modal" variation and the "submit" is required. I can add the "cancel" here and have both submit and cancel do the same thing, or keep as is, I am fine with either. But I think the real solution is to add an "informational / Close modal" to Hew.

Reply:
I got the idea from the GalleryModal. Is this modal different because it can make changes (`handleTrialUnselect`) ?